hyper/http11/connection.py

```diff
@@ -8,15 +8,18 @@
 import logging
 import os
 import socket
+import base64
 
 from .response import HTTP11Response
 from ..tls import wrap_socket
 from ..common.bufsocket import BufferedSocket
-from ..common.exceptions import TLSUpgrade
+from ..common.exceptions import TLSUpgrade, HTTPUpgrade
 from ..common.headers import HTTPHeaderMap
 from ..common.util import to_bytestring
 from ..compat import bytes
 
+from ..http20.connection import H2C_PROTOCOL
```

Comment:
I think it's technically the right place. 'h2c' is technically an ALPN token, even though it should never be used in ALPN, so `tls` would work fine. (And yeah, I did mean `tls`, I just didn't notice that Safari autocorrected it. :disappointed: )